decidim-proposals/app/views/decidim/proposals/admin/proposals/show.html.erb

```diff
@@ -74,4 +74,6 @@
   </div>
 </div>
 
-<%= render "decidim/proposals/admin/proposal_notes/proposal_notes" %>
+<% if allowed_to? :create, :proposal_note %>
```

Comment:
Shouldn't admins who can't create proposal notes be allowed to view existing notes?

Reply:
I kept the current behavior, and currently if you can't create proposal notes you can't view the current ones...